clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -518,7 +529,7 @@ else if (!future.isRetriable())
                     return false;
                 }
 
-                timer.sleep(rebalanceConfig.retryBackoffMs);
+                timer.sleep(retryBackoff.backoff(attempts++));
```

Comment:
The KIP says the following.

`However other places that utilize the retry.backoff.ms configuration will not be affected, notably the classes responsible for rebalancing. Therefore, it will not affect Connect worker or Consumer group rebalancing. 
`

But it seems that we are changing the backoff time related to rebalance here?

Reply:
Yes, true. Good point. I'll revert the code for this case.